pkgs/development/go-modules/generic/default.nix

```diff
@@ -198,7 +198,7 @@ go.stdenv.mkDerivation (
      ln -s "${dep.src}" "$d/src/${dep.goPackagePath}"
   ''
   ) goPath) + ''
-    export GOPATH="$d:$GOPATH"
+    export GOPATH=${lib.concatStringsSep ":" ( ["$d"] ++ ["$GOPATH"] ++ ["$PWD"] ++ extraSrcPaths)}
```

Comment:
@kamilchm our project also brings in our own packages and if the project itself is not added to the GOPATH it can't find them. do you think i'm doing something wrong?